encord/metadata_schema.py

```diff
@@ -268,22 +268,25 @@ def add_scalar(
         self,
         k: str,
         *,
-        data_type: Literal["boolean", "datetime", "number", "uuid", "text", "varchar", "string", "long_string"],
+        data_type: Union[
+            Literal["boolean", "datetime", "number", "uuid", "text", "varchar", "string", "long_string"], str
```

Comment:
this does remove type checking completely, would union literals or some user visible enum work?

Reply:
made an enum, so we got the best of both worlds.